ReactWindows/ReactNative.Shared.Tests/Bridge/JavaScriptModuleRegistryTests.cs

```diff
@@ -1,65 +1,18 @@
-﻿using Newtonsoft.Json.Linq;
-using NMock;
-using NMock.Syntax;
-using NUnit.Framework;
+﻿using NUnit.Framework;
 using ReactNative.Bridge;
-using ReactNative.UIManager;
-using ReactNative.UIManager.Events;
-using System.Threading;
 
 namespace ReactNative.Tests.Bridge
 {
     [TestFixture]
     public class JavaScriptModuleRegistryTests
     {
-        private MockFactory _mockFactory;
-        private Mock<IReactInstance> _mockReactInstance;
-        private JavaScriptModuleRegistry _registry;
-        private JavaScriptModuleRegistry.Builder _registryBuilder;
-
+        protected JavaScriptModuleRegistry _registry;
+        protected JavaScriptModuleRegistry.Builder _registryBuilder;
 
         [SetUp]
-        public void SetUp()
+        public virtual void SetUp()
         {
             _registryBuilder = new JavaScriptModuleRegistry.Builder();
-            _mockFactory = new MockFactory();
-            _mockReactInstance = _mockFactory.CreateMock<IReactInstance>();
-        }
-
-        [Test]
-        public void InvokesReactInstanceWhenFetchedModuleIsCalled()
```

Comment:
My latest push adds the InvokesReactInstanceWhenFetchedModuleIsCalled and ThrowsWhenUnkonwModuleRequested to the ReactNative.Tests using the implementations that existed before they were converted to NMock...